config/jobs/kubernetes/sig-release/cip/cloudbuilder.yaml

```diff
@@ -0,0 +1,20 @@
+promoter:
+  setup.sh: |
+    #!/bin/sh
+    set -o errexit
+    set -o nounset
+    set -o pipefail
+    set -o xtrace
+
+    shopt -s dotglob
+
+    git \
+      clone https://sigs.k8s.io/${_REPO} \
+      --depth 1 \
+      --branch $BRANCH_NAME \
+      ${_GOPATH}/src/sigs.k8s.io/${_REPO}
+
+    # Get golangci-lint; install into /workspace/bin/golangci-lint.
+    pushd go
+    curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh \
```

Comment:
pipe->bash always kinda scares me, I see they recommend it on https://github.com/golangci/golangci-lint, but this also feels like life before modules

what if we used a `golang:1.12` image and then did something like `go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.17.1` to use a well known version?